yjit/src/cruby.rs

```diff
@@ -301,6 +303,90 @@ pub struct rb_cref_t {
     _marker: core::marker::PhantomData<(*mut u8, core::marker::PhantomPinned)>,
 }
 
+pub struct Flags(c_uint);
```

Comment:
The name `Flags` is too generic IMO. Maybe `CallFlags`, with a comment explaining that this is used to describe call sites.

Reply:
Done